server/main.js

```diff
@@ -1,35 +1,35 @@
 // explicitly import all the stuff from lib/ since mainModule skips autoloading
 // things
-import '../imports/lib/config/accounts.js';
-import '../imports/lib/models/proptypes.js';
-import '../imports/lib/models/users.js';
-import '../imports/lib/roles.js';
+import '../imports/lib/config/accounts';
+import '../imports/lib/models/proptypes';
+import '../imports/lib/models/users';
+import '../imports/lib/roles';
 
 // Register migrations
-import '../imports/server/migrations/all.js';
+import '../imports/server/migrations/all';
 
 // Other stuff in the server folder
-import '../imports/server/accounts.js';
-import '../imports/server/announcements.js';
-import '../imports/server/ansible.js';
-import '../imports/server/api-init.js';
-import '../imports/server/api_keys.js';
-import '../imports/server/chat.js';
-import '../imports/server/feature_flags.js';
-import '../imports/server/fixture.js';
-import '../imports/server/git_revision.js';
-import '../imports/server/guesses.js';
-import '../imports/server/hunts.js';
+import '../imports/server/accounts';
+import '../imports/server/announcements';
+import '../imports/server/ansible';
+import '../imports/server/api-init';
+import '../imports/server/api_keys';
+import '../imports/server/chat';
+import '../imports/server/feature_flags';
+import '../imports/server/fixture';
+import '../imports/server/git_revision';
+import '../imports/server/guesses';
+import '../imports/server/hunts';
 import '../imports/server/migrations-run.js'; // runs migrations
```

Comment:
did you also want to change this line?

Reply:
Yes - my find-replace was kind of coarse, I guess :joy: